cmake.diff

```diff
@@ -0,0 +1,15 @@
+diff --git a/CMakeLists.txt b/CMakeLists.txt
+index 38ff055..b1b8867 100644
+--- a/CMakeLists.txt
++++ b/CMakeLists.txt
+@@ -44,6 +44,10 @@ else ()
+ 	set(PROJECT_VERSION_PATCH "0")
+ endif ()
+ 
++set(PROJECT_VERSION_MAJOR "0")
++set(PROJECT_VERSION_MINOR "6")
++set(PROJECT_VERSION_PATCH "0")
```

Comment:
why is this necessary?

Reply:
because of the bug mentioned in the manifest: https://github.com/slowmoVideo/slowmoVideo/issues/200

Reply:
so why not instead make sure it take the "else" branch? You talk about being easy on maintenance elsewhere, but make a patch that makes it harder since it would need to be updated each time if upstream never fix the issue.

Reply:
> so why not instead make sure it take the "else" branch?

I can't answer that question. I just wanted to make it work with little to no knowledge about cmake. I'm happy if you produce a better patch!